organizational/conduct.rst

```diff
@@ -0,0 +1,71 @@
+Code of Conduct
+===============
+
+Hacker Society [#hacsoc]_ is, as Tim wrote in :doc:`hacsoc`, about an open
+society:
+
+  An open society promotes freedom of thought and expression, personal
+  responsibility, individualism, cooperation, and equality of all citizens.
+
+As such, we expect that our members behave in a way that allows us to achieve
+this.  Participants in HacSoc events are expected to treat each other with
+dignity and respect for each other's perspectives, experiences, and knowledge.
+In particular, participants should recognize that people of all skill levels are
+welcome, and should always be willing to help others learn.
+
+We want Hacker Society to be a place where people can be comfortable regardless
+of gender, gender identity and expression, sexual orientation, disability,
+physical appearance, body size, race, age, or religion.  As such, we do not
+tolerate harassment.  This includes, but is not limited to:
+
+- Rude or unwelcome comments
```

Comment:
This should be clarified. I might find it unwelcome for someone to disparage my favorite editor, vim, I might even find it rude! However, it should be allowed. This is a silly example but one of the goals of an open society is to have freedom of thought and expression. This is necessarily in tension with freedom from rude or unwelcome comments. However, in the case of hacker society this is easily solve: comments attacking an individual on the grounds of: gender, gender identity, gender expression, attire, sexual orientation, disability, physical appearance, body size, body type, race, age, or religion are considered rude and unwelcome.

We should strive for politeness and civil discourse but we should also strive to make all welcome. Even those with unpopular ideas or thoughts. So when does an unpopular idea cross the line? To me this happens when it is an attack on an intrinsic characteristics of a person. Here is an example of an idea which attacks a *non-intrinsic* characteristic: I hate how "Joe" is so bad at using email he always replies on top instead of on bottom. The way "Joe" uses email is not intrinsic to Joe an may be violating other established community norms. However, saying: I hate how Joe always wears yellow shirts so ugly! That is saying something about an intrinsic to Joe their preference for yellow shirts. The former is allowed the latter would be viewed as unwelcome. In the same way saying: I love how Joe always uses vim, is fine but saying: I love how Joe always wears green it is so hot, is not fine.

This is a really tricky balance. In my views the goals are:

1. A welcoming environment
2. A non-sexualized environment
3. A free environment where people can explore ideas

Freedom is necessarily subservient to the other two. If not than the loudest and crassest have free reign. A community has goals and norms and an open society can have both and still be open. Open for who? All who want to engage with ideas not engage with attacking personal characteristics of other community members.